LEGO1/omni/include/mxautolock.h

```diff
@@ -0,0 +1,17 @@
+#ifndef MXAUTOLOCK_H
+#define MXAUTOLOCK_H
+
+#include "mxcriticalsection.h"
+
+#define AUTOLOCK(CS) MxAutoLock lock(&CS);
```

Comment:
Skip the semicolon, so users need to add it.
```suggestion
#define AUTOLOCK(CS) MxAutoLock lock(&CS)
```